docs/en/jump-start-extdev-plasma-us1.md

```diff
@@ -1,12 +1,12 @@
 ---
-id: jump-start-extdev-plasma-us1
+id: jump-start-extdev
```

Comment:
Lets not mess with the page IDs so we don't have to fix the inevitable broken references.